keps/sig-network/563-dual-stack/README.md

```diff
@@ -1416,36 +1410,21 @@ fields of API types, flags, etc.?**
 
 * **What are the SLIs (Service Level Indicators) an operator can use to determine 
 the health of the service?**
-  - [ ] Metrics
-    - Metric name:
-    - [Optional] Aggregation method:
-    - Components exposing the metric:
-  - [ ] Other (treat as last resort)
-    - Details:
+  Dual stack networking is a functional addition, not a service with SLIs.
```

Comment:
I added this line based on what appeared to be a lack of SLI information; please verify or correct.

Reply:
Also, based on the feedback in https://github.com/kubernetes/enhancements/pull/2325/files#r568829066 we need to fix the command line example.

Reply:
kubectl get services --all-namespaces -ogo-template='{{range .items}}{{.spec.ipFamilyPolicy}}{{"\n"}}{{end}}' | grep -v SingleStack